usernames.sql

```diff
@@ -0,0 +1 @@
+copy reserved_usernames (name) from '/tmp/usernames.txt';
```

Comment:
Why is it in `/tmp`?
Shouldn't it be in e.g. current dir?

Reply:
No, this for some reason isn't readable by postgresql. Plus, postgresql requires an absolute path, and copying to /tmp is cleaner.